app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -56,22 +56,12 @@ def save_proofing_components
         ProofingComponent.create_or_find_by(user: current_user).update(component_attributes)
       end
 
-      # @param [DocAuth::Response,
-      #   DocumentCaptureSessionAsyncResult,
-      #   DocumentCaptureSessionResult] response
```

Comment:
the return value of this method didn't change, let's keep this YARD type annotation?

Reply:
Updated.